src/components/wallet-manager/index.test.tsx

```diff
@@ -162,46 +133,4 @@ describe('WalletManager', () => {
 
     expect(updateConnector).toHaveBeenCalledWith(Connectors.Metamask);
   });
-
-  describe('mapState', () => {
```

Comment:
Where should the state / reducer / action related test coverage be moved to? Should they be move to the ../store directory or would they stay in this file?

Reply:
Ideally we should test:
- action creators in store folder (unit test)
- reducers in store folder (unit test)
- testing sagas in store folder (unit test)

and if we want we can test the integrations between the component and the store.